virttest/utils_test/__init__.py

```diff
@@ -2200,6 +2206,32 @@ def unload_stress(stress_type, params, vms=None, remote_server=False):
                    remote_server=remote_server).unload_stress()
 
 
+def prepare_profile(test, fpath, pat, repl):
+    """
+    This is to prepare client profile to be run on client.
+    :param test: kvm test object
+    :param fpath: profile to be run
+    :param pat: pattern to be replaced
+    :param repl: replacement strings includes server_ip, duration, threads etc.
+    :return: Edited profile which is ready to be run on client guest
+    :raise: TestFail: raised if unable to edit the given profile
```

Comment:
this seems to be mismatching with what is actually done. `TestError` is raised actually.